data/styles/vehicle/include/Icons.mapcss

```diff
@@ -124,6 +124,10 @@ node|z14-[aeroway=aerodrome]::int_name,
 
 /*check offset int_name*/
 
+/* 3.2 Road Icons */
+node|z19-[highway=traffic_signals],
```

Comment:
For vehicle style, does it make sense to show traffic lights earlier? Will they be visible at all on mobile devices when the camera is up at the city speed of 30-50 mph?